open-api.yaml

```diff
@@ -1476,17 +1503,31 @@ paths:
                     type: string
                     example: '["title", "overview"]'
                     default: '[]'
+                highlightPreTag:
```

Comment:
I'm questioning myself about the usage of `highlightPreTag` or `highlightPostTag` alone. 
Imagine a user going to highlight their search.

The dev called the `search` method using a different highlighter tag, but it did not close or vice-versa, it will have a not-expected result.

```rb
client.index('movies').search("batman", highlightPreTag: "<mark>")
#=> [{..., text: "<mark>batman</em>" }]

client.index('movies').search("batman", highlightPostTag: "</mark>")
#=> [{..., text: "<em>batman</mark>" }]

client.index('movies').search("batman", highlightPostTag: "$$$$")
#=> [{..., text: "<em>batman$$$$" }]
```

My question is, it is OK to allow users to make this kind of request (using just one or another)?
I'm thinking about the possibility to raise an exception if the user wants to use one or another alone.

But I would like to grab some opinions on this subject.
FYI: @gmourier

Reply:
I don't see a direct problem with letting them do this for now as I imagine users will specify both fields most of the time.

I see a case for not forcing both to be set for the future. If we add an index setting that allows custom default values to be set, a frontend client might only want to change one or the other at search time so as not to override one of the two already set defaults.

Also, today if a user wants to add a custom class to `<em>` or something afterward without modifying the `highlightPostTag` it will not be possible if we force both to be defined.

I don't think it's a problem to leave it the way it is for now but if you are against it and think it's better for the UX, I'm with you. 👍  

@ManyTheFish You may have an opinion on that since you are on it